diagnostic.md

```diff
@@ -5,74 +5,82 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+To process requests and send them back to the client/front-end
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model layer is used by the controller ot fetch data
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller communicates with the model
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Accessing the Model directly from views does not separate concern and goes against the MVC pattern. 
```

Comment:
So the reason we don't use rails views is because we build SPAs which handle view states on the front-end. This allows us to lessen the amount of requests we make to the server.